package.json

```diff
@@ -50,16 +50,17 @@
     "jeet": "^7.0.0",
     "normalize.css": "^7.0.0",
     "picturefill": "^3.0.2",
-    "replace-in-file": "^2.5.0",
+    "replace-in-file": "^3.0.0-beta.2",
```

Comment:
why beta?

Reply:
because yarn did that with `yarn upgrade --latest`... should I downgrade again?